wr-cli/parser.py

```diff
@@ -0,0 +1,103 @@
+from enum import Enum
+
+
+class ParserError(Enum):
+    EMPTY_STRING = 0
+    KEY_VALUE_PAIR = 1
+
+
+class Parser:
+    def _separate_tokens(self, user_input):
+        """
+        Helper to separate parts of the string according to some rules.
+
+        Parameters
+        ----------
+        user_input : string
+            input string
+
+        Returns
+        -------
+        list
+            array of separated string tokens
+        """
+        arr_user_string = user_input.split()
+        wrap_character= ""
+        stack = []
+        final_arr_user_string = []
+        for token in arr_user_string:
+            # keep the original token for comparisons, modified token for
+            # inserting into stack
+            modified_token = token.replace("\"", "").replace("\'", "")
```

Comment:
My bad. There's apparently a string function `strip()` that'll be better for this.